terraso_backend/apps/graphql/schema/commons.py

```diff
@@ -62,6 +62,24 @@ def mutate(cls, root, info, input):
             return cls(errors=[{"message": str(error)}])
 
 
+class BaseAdminMutation(BaseMutation):
+    class Meta:
+        abstract = True
+
+    @classmethod
+    def mutate(cls, root, info, input):
+        user = info.context.user
+
+        if not user or not user.is_authenticated or not user.is_superuser:
```

Comment:
question: what do we need the superuser check for?

Reply:
@paulschreiber I think so, for the current available operations added as part of this PR it makes sense to just let superuser access them